Assets/Runtime/Core/AUIScreenController.cs

```diff
@@ -190,7 +190,7 @@ public void Show(IScreenProperties props = null)
             HierarchyFixOnShow();
             OnPropertiesSet();
 
-            if (!gameObject.activeSelf)
+            if (!gameObject.activeSelf || AnimOut != null)
```

Comment:
No need to close previous animation manually in this case?

Reply:
it's, already included to `DoAnimation()`